app.js

```diff
@@ -1,6 +1,7 @@
 //=====================================
 //INITIALISATION
 //=====================================
+require("dotenv").config();
```

Comment:
How would this line affect pre-existing environment variables? If I have a permanent local variable on the deployed server with `MY_PLATFORM=server`, would that get overwritten?

Reply:
Yes, I think so. You would just put `MY_PLATFORM=server` into your `.env` file instead.